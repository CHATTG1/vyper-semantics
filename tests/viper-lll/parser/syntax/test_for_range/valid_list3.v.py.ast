%pgm(, ,
  %fdecl(%@public, foo, , %void,
    %assign(%var(x), 5)
    %forrange(i, %var(x), %binop(+, %var(x), 10),
      %pass))
)
