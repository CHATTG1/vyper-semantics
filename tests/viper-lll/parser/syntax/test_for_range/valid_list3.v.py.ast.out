$%seq ( $return ( 0 , $lll ( $%seq ( $%seq ( $mstore ( 28 , $calldataload ( 0 ) ) , $mstore ( 32 , 1461501637330902918203684832716283019655932542976 ) , $mstore ( 64 , 170141183460469231731687303715884105727 ) , $mstore ( 96 , -170141183460469231731687303715884105728 ) , $mstore ( 128 , 1701411834604692317316873037158841057270000000000 ) , $mstore ( 160 , -1701411834604692317316873037158841057280000000000 ) ) , $if ( $eq ( $mload ( 0 ) , 3264763256 ) , $%seq ( $pass , $assert ( $iszero ( $callvalue ) ) , $mstore ( 320 , 5 ) , $repeat ( 352 , $mload ( 320 ) , 10 , $%seq ( $pass ) ) , $stop ) ) ) , 0 ) ) )
